README.md

```diff
@@ -18,8 +18,20 @@ composer require wyrihaximus/list-classes-in-directory
 # Usage
 
 ```php
+
+use function WyriHaximus\listClassesInDirectories;
+use function WyriHaximus\listClassesInDirectory;
+
 // $classes now contains a list of full qualified class names from __DIR__
 $classes = listClassesInDirectory(__DIR__);
+
+// you can also get q list of classes from multiple directories 
```

Comment:
Assuming this should be `get a list`?

Reply:
i just switched from azerty to a qwerty keyboard .. ugh

Reply:
Oef getting used to that takes time, good luck!